### PR TITLE
Add Regions used during the background estimate validation

### PR DESCRIPTION
**Pull request by dapparu**
- Add of the class Regions composed of a set of histograms used during the validation of the background estimate method
- Add of functions used to estimate the background in signal region

### CODE REVIEW

File: Analyzer/interface/Regions.h [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+float K = 1;
+float C = 1;
+
+// Scale the 1D-histogram given to the unit 
+void scale(TH1F* h){
+    h->Scale(1./h->Integral(0,h->GetNbinsX()+1));
+}
+
+// class using to definite signal and control regions. 
+class region{
```
Comment (tvami) [inline Analyzer/interface/Regions.h:24]: Class names should be upper case :P 
rule 2.6
https://cms-sw.github.io/cms_coding_rules.html

Comment (dapparu) [reply Analyzer/interface/Regions.h:24]: Oooops... I'll fix it. (:

File: Analyzer/plugins/Analyzer.cc [hunk 2]
```diff
@@ -12,35 +12,9 @@
...
-// - 18p8: Add postPreselection plots
-// - 19p0: One try with TOF
-// - 19p1: Change mass binning, remove massT cut
-// - 19p3: Simplify probQ cut, change mini-iso def
-// - 19p4: Change mini-iso binning
-// - 19p5: use charged iso in cutflow, dont cut away out of bound probs, only in preselection
-// - 19p6: intro CutFlowEta and PerGenID
-// - 19p7: intro NumEvents and HSCPCandidateType, for comparrison, put back EoP cut and TkIso cut (will remove in 19p8)
+//v20
+// - v20.1 add Regions used to vlaidate the background estimate method
```
Comment (tvami) [inline Analyzer/plugins/Analyzer.cc:16]: We need to figure out a better versioning system when we both develop... I'm actually at 21.0 now (it's my fault, I didnt PR my changes)

Comment (dapparu) [reply Analyzer/plugins/Analyzer.cc:16]: What's your proposal?

Comment (tvami) [reply Analyzer/plugins/Analyzer.cc]: Version by names maybe? v20p1_Dylan? while I have v20p9_Tamas and then we somehow call the sum of the two v21 in the common code? Not sure....

**Pull request merged**